### PR TITLE
Try to fix trove_arbitrum_trades

### PR DESCRIPTION
**Pull request by Hosuke**
Not sure the exact cause for the issue:
https://dune.com/queries/2042429?d=11&category=abstraction
https://discord.com/channels/757637422384283659/999683200563564655/1078530354194960535
image

The model works fine for spark sql though.

### CODE REVIEW

File: models/trove/arbitrum/trove_arbitrum_trades.sql [hunk 1]
```diff
@@ -63,12 +79,14 @@ select
     project_contract_address,
     cast(null as varchar(5)) as aggregator_name,
     cast(null as varchar(5)) as aggregator_address,
-    tx_hash,
+    mp.tx_hash,
     mp.block_number,
     tx.from as tx_from,
```
Comment (belen-pruvost) [inline models/trove/arbitrum/trove_arbitrum_trades.sql:84]: Hey @Hosuke! `from` is a reserved word on DuneSQL so it needs to be escaped when it's used for selecting a column. Changing this to be tx.`from` should do the trick 🤞

**Review by belen-pruvost — CHANGES_REQUESTED** [review]

File: models/trove/arbitrum/trove_arbitrum_trades.sql (hunk 1 above)
Comment (Hosuke) [reply models/trove/arbitrum/trove_arbitrum_trades.sql:84]: like this?
```suggestion
     tx.`from` as tx_from,
```

Comment (belen-pruvost) [reply models/trove/arbitrum/trove_arbitrum_trades.sql]: Exactly 👍

**Review by belen-pruvost — APPROVED** [review]
This looks good to me! I'll tag the team that owns Spellbook to give it the final thumbs up + merge

**Review by 0xRobin — APPROVED** [review]
thank you!

**Pull request merged**